packages/channels/botpress-channel-messenger/README.md

```diff
@@ -47,11 +47,17 @@ npm i @botpress/channel-messenger
 
 [Config template](https://github.com/dmytropanontko/botpress/blob/docs/messenger-tutorial/packages/channels/botpress-channel-messenger/config.json)
 
+`"freezeProfile"` - if set, do not update Messenger Profile on bot restart. `Default: false`
+
+`"custom"` - convenience config for globally storing any custom bot-specific settings
```

Comment:
This is for any configuration data that I wouldn't want to hard code

Right now I am using it to store `pixel_id` that would fire in webviews across all the page

But it could store others things in a future, maybe substitution tokens, etc

Generally anything configured across all pages could go here

Reply:
Are these custom settings messenger-specific?

Reply:
I believe they are

Anything about Messenger webviews definitely is